feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/captive.c

```diff
@@ -386,22 +468,25 @@ void vif_captive_portal_set(const struct schema_Wifi_VIF_Config *vconf, char *if
 
 		}
 
-		char file_path[128];
-		struct stat st = {0};
-		if (stat(path, &st) == -1) {
-			mkdir(path, 0755);
-		}
-
 		else if (strcmp(opt, "splash_page_logo") == 0) {
 			sprintf(file_path,"%s%s",path,"TipLogo.png");
 			blobmsg_add_string(&cap, "splash_page_logo", value);
-			splash_page_logo(file_path,value);
+			if(strcmp(splash_logo,value) !=0)
+			{
+				splash_page_logo(file_path,value);
+			}
+			strncpy(splash_logo, "0", strlen(splash_logo));
 		}
 
 		else if (strcmp(opt, "splash_page_background_logo") == 0) {
+
 			sprintf(file_path,"%s%s",path,"TipBackLogo.png");
 			blobmsg_add_string(&cap, "page_background_logo", value);
-			splash_page_logo(file_path,value);
+			if(strcmp(back_image,value) !=0)
+			{
```

Comment:
Can we follow Kernel coding style guideline?  Please refer section 3 of the document (Placing braces and whitespaces). Please change at other places as well.

Reply:
Sure, I'll do that